### PR TITLE
Add SSR experiments to a4a eid

### PR DESCRIPTION
**Pull request by jridgewell**
This adds the currently set SSR experiments to the a4a eid.

A small hiccup with this setup is that the SSR experiments are usually just boolean values (or small enums) that don't have a unique experiment id. I've created a pseudo id by concatenating the proto field id (eg, `25`) with a padded value (eg, `0`) to create the exp id (`2500`).

Re: #30504
Re: cl/335090010

### CODE REVIEW

File: extensions/amp-a4a/0.1/amp-a4a.js
Comment (powerivq) [inline extensions/amp-a4a/0.1/amp-a4a.js:2318]: Should we create real exp id for these ssr features? I am not sure this pseudo id is permissible.

**Comment by jridgewell** [comment]
This is ready for review @powerivq @jeffkaufman

File: ads/google/a4a/utils.js
Comment (lannka) [inline ads/google/a4a/utils.js:76]: Is mendel a valid concept here?

File: ads/google/a4a/utils.js
Comment (lannka) [inline ads/google/a4a/utils.js:216]: is it a typo? aexp vs aexps?

**Review by powerivq — APPROVED** [review]

File: ads/google/a4a/utils.js
Comment (jeffkaufman) [reply ads/google/a4a/utils.js:216]: not a typo, the param is called "aexp" internally.

Comment (lannka) [reply ads/google/a4a/utils.js:216]: ok, i suggest we make variable name the same to avoid mistakes.

File: ads/google/a4a/utils.js
Comment (jridgewell) [reply ads/google/a4a/utils.js:76]: I'm not sure I understand?

File: ads/google/a4a/utils.js
Comment (jridgewell) [reply ads/google/a4a/utils.js:216]: Done.

**Comment by jridgewell** [comment]
Can I get one more approval before merging? @lannka or @jeffkaufman?

**Review by lannka — APPROVED** [review]

**Review by jeffkaufman — APPROVED** [review]

**Pull request merged**
